codechecker_lib/arg_handler.py

```diff
@@ -433,14 +428,17 @@ def handle_plist(args):
 
     try:
         items = [(plist, args, context) for plist in os.listdir(args.directory)]
-        pool.map_async(consume_plist, items, 1).get(float('inf'))
+        pool.map_async(consume_plist, items, 1)
```

Comment:
I do not think this should be removed (timeout is set to "infinite") see out other [usage of map_async](https://github.com/Ericsson/codechecker/blob/eeac2d6a1b1188dce6ef65bcc4a82faa545a5f7e/codechecker_lib/analysis_manager.py#L198-L209).